src/Core/TileMesh.js

```diff
@@ -14,6 +14,7 @@ import { geoidLayerIsVisible } from 'Layer/GeoidLayer';
  */
 class TileMesh extends THREE.Mesh {
     #_tms = new Map();
+    #visible = true;
```

Comment:
why make it private if it can be get and set ?

Reply:
to avoid `this._visible and this.visible`

Reply:
oh ok I thought we could write getters on `this.visible` directly but apparently not